tests/impl.cpp

```diff
@@ -6081,7 +6081,17 @@ result_t test_mm_stream_si32(const SSE2NEONTestImpl &impl, uint32_t iter)
 
 result_t test_mm_stream_si64(const SSE2NEONTestImpl &impl, uint32_t iter)
 {
-    return TEST_UNIMPL;
+    const int64_t a = (const int64_t) impl.mTestInts[iter];
+#if defined(__x86_64__) || defined(__i386__) || defined(__LLP64__)
+#if defined(_WIN32) || defined(__APPLE__) || defined(__linux__)
```

Comment:
It is not necessary to enumerate the operating systems in favor of ABI.

Reply:
Sorry for answering late. I was taking the exam until this week.
I still don't understand the request.
Should I amend in the following way?

```c
#if defined(__x86_64__) || defined(__i386__)
#if defined(_WIN32)  || defined(__LLP64__)
    long long p[1];
#else
    int64_t p[1];
#endif
#else
    int64_t p[1];
#endif
```

Reply:
Alternatively, consider the following:
```c
#if (defined(__x86_64__) || defined(__i386__)) && (defined(_WIN32) || defined(__LLP64__))
    long long p[1];
#else
    int64_t p[1];
```

Reply:
I checked the test. The test can't successfully build on the machine which runs Linux on x86.

Reply:
we should use `long long p[1]` on Linux on x86 machine

Reply:
@jserv is the current fashion meets your reuqest?